3ds2-example/frontend/components/sessions/components.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link rel="stylesheet" href="../../style.css">
+
+    <title>Components Integration - sessions flow</title>
+  </head>
+  <body>
+    <header>
+      <h1>Components Integration - sessions flow</h1>
+      <a href="/">Menu</a>
+    </header>
+    <main>
+      <section>
+      <p>This example is a component integration with sessions flow.  For more information on how to configure this integration please refer to the <a href="https://docs.adyen.com/online-payments/build-your-integration/additional-use-cases/advanced-flow-integration/?platform=Web&integration=Components&version=5.53.2" target="_blank">documentation.</a></p>
+    </section>
+    
+    <section>
+      <p>Please note for 3DS2 only native flow is supported with /sessions flow so if you wish to use redirect flow please use advanced flow.</p>
```

Comment:
Rewrite suggestion: The native flow is compatible with `/sessions` flow for 3DS2. If you intend to use the redirect flow, use the advanced flow (add a link to the advanced flow /folder))

Reply:
✅